### PR TITLE
RHcloud insights rake commands

### PR DESCRIPTION
**Pull request by spusateri**
Passing locally, however I think the assertions can be added upon and improved. Would checking whether a remediation is present be outside the scope of this test?

### CODE REVIEW

**Comment by jameerpathan111** [comment]
@spusateri can you resolve the conflicts? My pr is merged now.

File: tests/foreman/cli/test_rhcloud_inventory.py [hunk 1]
```diff
@@ -106,6 +106,11 @@ def test_positive_inventory_recommendation_sync():
```
Comment (jameerpathan111) [inline tests/foreman/cli/test_rhcloud_inventory.py:106]:
```suggestion
:CaseAutomation: Automated
```

File: tests/foreman/cli/test_rhcloud_inventory.py [hunk 2]
```diff
@@ -106,6 +106,11 @@ def test_positive_inventory_recommendation_sync():
 
     :CaseLevel: System
     """
+    org, ak = organization_ak_setup
+    cmd = f'organization_id={org.id} foreman-rake rh_cloud_insights:sync'
+
+    result = rhcloud_sat_host.execute(cmd)
+    assert result.status == 0
```
Comment (jameerpathan111) [inline tests/foreman/cli/test_rhcloud_inventory.py:113]: Can you add assertion for `Synchronized Insights hosts hits data` in `result.stdout`

File: tests/foreman/cli/test_rhcloud_inventory.py [hunk 3]
```diff
@@ -85,8 +85,8 @@ def test_positive_inventory_generate_upload_cli(
         assert hosts_count == local_file_data['slices_counts'][slice_name]
 
 
-@pytest.mark.stubbed
-def test_positive_inventory_recommendation_sync():
+# @pytest.mark.stubbed
+def test_positive_inventory_recommendation_sync(organization_ak_setup, rhcloud_sat_host):
```
Comment (jameerpathan111) [inline tests/foreman/cli/test_rhcloud_inventory.py:89]:
```suggestion

@pytest.mark.tier3
def test_positive_inventory_recommendation_sync(
    set_rh_cloud_token,
    organization_ak_setup,
    rhcloud_registered_hosts,
    rhcloud_sat_host,
):
```

File: tests/foreman/cli/test_rhcloud_inventory.py [hunk 4]
```diff
@@ -85,8 +85,8 @@ def test_positive_inventory_generate_upload_cli(
         assert hosts_count == local_file_data['slices_counts'][slice_name]
 
 
-@pytest.mark.stubbed
-def test_positive_inventory_recommendation_sync():
+# @pytest.mark.stubbed
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_rhcloud_inventory.py:88]: This should be removed, instead of a comment.

**Review by latran — CHANGES_REQUESTED** [review]
Pending changes from other's suggestion and a test result.

**Comment by latran** [comment]
@jameerpathan111 can you take over this PR?

**Comment by Griffin-Sullivan** [comment]
Looks like this was already done in https://github.com/SatelliteQE/robottelo/pull/9330. We should go ahead and close this out.

**Pull request closed without merging**